community/cypher/frontend-2.3/src/main/scala/org/neo4j/cypher/internal/frontend/v2_3/helpers/NonEmptyList.scala

```diff
@@ -298,13 +298,15 @@ sealed trait NonEmptyList[+T] {
 }
 
 final case class Fby[+T](head: T, tail: NonEmptyList[T]) extends NonEmptyList[T] {
-  def tailOption: Option[NonEmptyList[T]] = Some(tail)
-  def hasTail: Boolean = true
-  def isLast: Boolean = false
+  override def tailOption: Option[NonEmptyList[T]] = Some(tail)
+  override def hasTail: Boolean = true
+  override def isLast: Boolean = false
+  override def toString = s"${head.toString}, ${tail.toString}"
```

Comment:
No the default implementation includes the class name, and the text `Fby` as well as `Last` is unhelpful and confusing inside plan descriptions.